sources/javascripts/concat/base.js

```diff
@@ -569,6 +569,18 @@
     }
   };
 
+  // ===========================================================================
+  // Opens product admin view on product image click
+  // ===========================================================================
+
+  var handleProductImageClick = function(url) {
+    if (editmode()) {
+      $('.product-content .item-top').click(function() {
```

Comment:
`item-top` is quit random target. Let's add `product-image` class to product image div.